composer.json

```
@@ -20,7 +20,7 @@
         "league/commonmark": "^2.2",
         "league/csv": "^9.0",
         "league/glide": "^1.1 || ^2.0",
-        "maennchen/zipstream-php": "^2.2",
+        "maennchen/zipstream-php": "^3.1",
```

```suggestion
        "maennchen/zipstream-php": "^2.2 || ^3.1",
```

As long as there's no breaking changes in 3.1, which it doesn't look like there is, I think we should probably allow for either version to be installed since Statamic still supports PHP 8.0.